src/main/java/frc/robot/subsystems/CommandSwerveDrivetrain.java

```diff
@@ -56,11 +56,17 @@ public class CommandSwerveDrivetrain extends TunerSwerveDrivetrain implements Su
   public final double DRIVE_BASE_RADIUS =
       Math.max(
           Math.max(
-              Math.hypot(TunerConstants.FrontLeft.LocationX, TunerConstants.FrontLeft.LocationY),
-              Math.hypot(TunerConstants.FrontRight.LocationX, TunerConstants.FrontRight.LocationY)),
+              Math.hypot(
+                  TunerConstants_Comp.FrontLeft.LocationX, TunerConstants_Comp.FrontLeft.LocationY),
```

Comment:
we should take these values from a function on the drivetrain, or otherwise configure it at runtime